src/res/props-grammar.txt

```diff
@@ -2,12 +2,15 @@ StyleSet <- S Style* EOF
 S <- (Space / Comment)*
 Space <- [ \t\r\n]
 Comment <- '/*' (!'*/' .)* '*/' 
+GID <- NestedNamedID / ID Space ID / ID
+NestedNamedID <- NamedID Space ID / ID Space NamedID
+NamedID <- ID ':' ID
```

Comment:
Part to the left from : should be Namespace, not ID. It is true, that currently, these Namespace is an alias for ID, but it has a different semantics. Also, in the future, Namespace or ID may change.